### PR TITLE
Fix "writable-strings" warnings

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: common/combuf.cpp [hunk 1]
```diff
@@ -832,31 +832,31 @@ void CommBufferClass::Mono_Debug_Print(int refresh)
...
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�                                      �                                      �\n");
+        Mono_Printf("�������������������������������������������������������������������������������");
```
Comment (ghost) [inline common/combuf.cpp]: ahh :(

Comment (OmniBlade) [reply common/combuf.cpp]: Yeah, if you could revert that kind of change that would be great, if anything the file needs changing to utf-8 properly so it uses the correct box drawing glyphs, but not until what if anything is decided over how the mono stuff is going to be used.

Comment (ghost) [reply common/combuf.cpp]: should be fixed now

**Comment by hifi** [comment]
Can these be made build errors so we don't reintroduce them later by accident?

**Comment by OmniBlade** [comment]
I believe that would need the `-Werror=write-strings` adding to the compiler flags for gcc and probably clang and `/Zc:strictStrings` for MSVC. Other than that this PR looks good to merge to me.

**Review by OmniBlade — APPROVED** [review]
Tested this a while ago, I don't think adding the compiler warnings should hold back this from being merged.

**Pull request merged**